chart/values.yaml

```diff
@@ -101,3 +101,6 @@ resources: {}
 tls: ingress
 
 systemDefaultRegistry: ""
+
+# Set to use the packaged system charts(available as of v2.3.0)
+useBundledSystemChart: true
```

Comment:
Need confirmation from @deniseschannon but it seems to default to false.(https://github.com/rancher/rancher/blob/master/pkg/settings/setting.go#L67)

Reply:
updated to false.